.github/workflows/ci.yml

```diff
@@ -79,13 +83,15 @@ jobs:
           my-project \
           --UNSAFE \
           --data 'full_name=Bruno Alla' \
-          --data 'email=test@example.com' \
-          --data 'github_username=dummy' \
+          --data 'email=action@github.com' \
+          --data 'github_username=actions-user' \
           --data 'project_name=${{ matrix.extra_options.project_name }}' \
           --data 'project_slug=${{ matrix.extra_options.project_slug }}' \
           --data 'project_short_description=Just a great project' \
           --data 'open_source_license=MIT' \
           ${{ matrix.extra_options.value }} \
+          --data 'initial_commit=yes' \
+          --data 'add_me_as_contributor=yes' \
```

Comment:
Could we run it only on one of the combination instead of all of them please? Would reduce change of rate limit